lib/resolvers/root.js

```diff
@@ -16,7 +16,7 @@ const _wrapResolver = (service, resolver) =>
         const entity = service.entities[fieldName]
         const responseKey = field.alias?.value || fieldName
 
-        response[responseKey] = resolver(service, entity, field)
+        response[responseKey] = parallel ? resolver(service, entity, field) : await resolver(service, entity, field)
```

Comment:
The promises are awaited via Promise.all by the GraphQL framework?

Reply:
Or is that done by you?

Reply:
I don't think that works

Reply:
```js
await Promise.all(enrichedFieldNodes.map(fieldNode => Promise.all(fieldNode.selectionSet.selections.map(async field => {
  const fieldName = field.name.value
  const entity = service.entities[fieldName]
  const responseKey = field.alias?.value || fieldName
  response[responseKey] = await resolver(service, entity, field)
}))))
```

Reply:
Adapted solution as discussed in our call